### ISSUE TITLE
Decide how to check observations with commanded 0 dither during acq 

### ISSUE THREAD

**Issue by jeanconn** [sot/starcheck#284]
For observations with monitor windows and observations approaching a dark cal, there can be acquisitions during the interval when commanded dither is 0 ampl or dither disable. Should starcheck us the previous dither (which may be the effective dither at the time of acquisition)?

For starcheck 13.0 it will be feeding proseco.get_acq_catalog with the commanded dither for these rare special-case observations.

**Comment by taldcroft** [comment]
Can this be closed?  It looks like the decision has been made e.g. in 21681 in MAY1319.

**Comment by jeanconn** [comment]
I don't think we should close this, because IIRC, you weren't happy with the 0 choice?  It is in the noise, but I think we should probably have a separate pad or some such because your point was that commanded 0 dither does not mean instantaneous 0 motion.  But super low priority.
